README.md

```diff
@@ -16,11 +16,25 @@ In this modified Cell Painting, there are five channels:
 
 ## Plate maps
 
-We applied this modified Cell Painting assay using the following plate design:
+We applied this modified Cell Painting assay using the following plate design for the first two plates:
 
-![treatment_dose_platemap.png](4.analyze-data/platemap_figures/treatment_dose_platemap.png)
+- Plate 1 = **localhost220512140003_KK22-05-198**
```

Comment:
What metadata is in this plate name? can it be changed for downstream naming conventions?

Reply:
The metadata associated with it would be if there are different culture or staining conditions on the plate like with `Plate 2` including F-actin adjusted in the name. In general, the plate names are pretty arbitrary for now.